src/github_runner_manager/manager/runner_scaler.py

```diff
@@ -120,47 +133,29 @@ def reconcile(self, quantity: int) -> int:
             quantity: The number of intended runners.
 
         Returns:
-            The Change in number of runners.
+            The Change in number of runners or reactive processes.
         """
         logger.info("Start reconcile to %s runner", quantity)
 
-        if self._reactive_config is not None:
-            logger.info("Reactive configuration detected, going into experimental reactive mode.")
-            return reactive_runner_manager.reconcile(
-                expected_quantity=quantity,
-                runner_manager=self._manager,
-                runner_config=self._reactive_config,
-            )
-
         metric_stats = {}
         start_timestamp = time.time()
 
         try:
-            delete_metric_stats = None
-            metric_stats = self._manager.cleanup()
-            runners = self._manager.get_runners()
-            logger.info("Reconcile runners from %s to %s", len(runners), quantity)
-            runner_diff = quantity - len(runners)
-            if runner_diff > 0:
-                try:
-                    self._manager.create_runners(runner_diff)
-                except MissingServerConfigError:
-                    logging.exception(
-                        "Unable to spawn runner due to missing server configuration, "
-                        "such as, image."
-                    )
-            elif runner_diff < 0:
-                delete_metric_stats = self._manager.delete_runners(-runner_diff)
+            if self._reactive_config is not None:
```

Comment:
I think in the long term it might be good to refactor the `RunnerScaler` class into `ReactiveRunnerScaler` and `RunnerScaler` (just a comment to trigger a discussion, feel free to resolve!)

Reply:
Maybe add a spike for this?